client/src/templates/Challenges/classic/editor-tabs.tsx

```diff
@@ -40,16 +40,25 @@ class EditorTabs extends Component<EditorTabsProps> {
       <div className='monaco-editor-tabs'>
         {/* eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call */}
         {sortChallengeFiles(challengeFiles).map(
-          (challengeFile: ChallengeFile) => (
-            <button
-              aria-expanded={visibleEditors[challengeFile.fileKey] ?? 'false'}
-              key={challengeFile.fileKey}
-              data-cy={`editor-tab-${challengeFile.fileKey}`}
-              onClick={() => toggleVisibleEditor(challengeFile.fileKey)}
-            >
-              {`${challengeFile.name}.${challengeFile.ext}`}{' '}
-              <span className='sr-only'>editor</span>
-            </button>
+          (
+            challengeFile: ChallengeFile,
+            index: number,
+            array: ChallengeFile[]
+          ) => (
+            <>
+              <button
+                aria-expanded={visibleEditors[challengeFile.fileKey] ?? 'false'}
+                key={challengeFile.fileKey}
+                data-cy={`editor-tab-${challengeFile.fileKey}`}
+                onClick={() => toggleVisibleEditor(challengeFile.fileKey)}
+              >
+                <span>{`${challengeFile.name}.${challengeFile.ext}`}</span>{' '}
+                <span className='sr-only'>editor</span>
+              </button>
+              {index !== array.length - 1 && (
+                <div className='editor-tab-separator' />
+              )}
```

Comment:
Funny enough, when you have added the operator made me sure that if we separate them indefinitely through a margin, is a better look than the current one we have.

This will remove the need for this logic and the surrounding CSS, but I will leave this to you and @ahmadabdolsaheb to decided.

Reply:
@Sboonny Definitely, adding a little space between each button would be a good solution as well, such as I did with the new toggle buttons on the Settings page. In fact, this is what I was originally going to do but I didn't want to make that decision on my own. If you and @ahmadabdolsaheb are OK with a permanent small margin between the buttons then I'll change it.

Reply:
Added permanent spacing between buttons.